### PR TITLE
Added vni field in VRF Yang for VxLAN L3 VNI Support #13456

### PR DESCRIPTION
**Pull request by tapashdas**
#### Why I did it
Added vni field in VRF Yang for VxLAN L3 VNI Support.

The VRF table schema as per EVPN HLD is below
https://github.com/sonic-net/SONiC/blob/master/doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

Addresses Issue #13456

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

**Comment by tapashdas** [comment]
@dgsudharsan
@srj102

**Review by dgsudharsan — CHANGES_REQUESTED** [review]
Please add UT, Updated sample_config_db.json and updated Configuration.md

File: src/sonic-yang-models/tests/yang_model_tests/tests_config/vrf.json [hunk 1]
```diff
@@ -28,5 +28,16 @@
                 }]
             }
         }
+    },
```
Comment (dgsudharsan) [inline src/sonic-yang-models/tests/yang_model_tests/tests_config/vrf.json:31]: @tapashdas  Can you please add one more test for negative scenario (out of range)?

**Review by dgsudharsan — CHANGES_REQUESTED** [review]

File: src/sonic-yang-models/tests/yang_model_tests/tests_config/vrf.json (hunk 1 above)
Comment (tapashdas) [reply src/sonic-yang-models/tests/yang_model_tests/tests_config/vrf.json:31]: done

**Review by dgsudharsan — APPROVED** [review]

**Review by prsunny — APPROVED** [review]

**Comment by keboliu** [comment]
@StormLiangMS  @dgsudharsan since the PR for 202211 has been merged, shall we remove the tag "request for 202211 branch?"
